modules/web/src/main/java/org/jboss/weld/module/web/logging/ServletLogger.java

```diff
@@ -97,4 +97,8 @@
     @Message(id = 718, value = "No EEModuleDescriptor defined for bean archive with ID: {0}. @Initialized and @Destroyed events for ApplicationScoped may be fired twice.", format = Format.MESSAGE_FORMAT)
     void noEeModuleDescriptor(Object beanArchiveId);
 
+    @LogMessage(level = Level.DEBUG)
+    @Message(id = 719, value = "When attempting to activate session context, an already active HttpSessionDestructionContext was detected. This is likely a leftover from previous sessions that ended exceptionally; this context will be terminated. ", format = Format.MESSAGE_FORMAT)
```

Comment:
@mkouba if you have an idea for better log msg, I am all ears

Reply:
Looks good but it's a bit too long. Maybe just "An active HttpSessionDestructionContext detected. This is likely a leftover from previous sessions that ended exceptionally; this context will be terminated."?

Reply:
+1, will adjust that